qcodes/instrument_drivers/american_magnetics/AMI430.py

```diff
@@ -787,6 +949,13 @@ def _adjust_child_instruments(
                 instrument.set_field(value, perform_safety_check=False,
                                      block=self.block_during_ramp.get())
 
+    def wait_while_all_axes_ramping(self) -> None:
+        axes = (self._instrument_x, self._instrument_y, self._instrument_z)
+        while all(
```

Comment:
```suggestion
        while any(
```

Reply:
This is what we also changed in our test